barretenberg/acir_tests/browser-test-app/src/index.ts

```diff
@@ -27,14 +27,14 @@ async function runTest(
   );
 
   const acirComposer = await api.acirNewAcirComposer(CIRCUIT_SIZE);
-  const proof = await api.acirCreateProof(
+  const [publicInputs, proofWithOutPublicInputs] = await api.acirCreateProof(
```

Comment:
Naming it like this is exposing the internal implementation details that a proof traditionally has both.
If we're presenting an API, let's name in a manner that would make sense to the consumer, in which case just `proofData` or `proof` is sufficient.
Let's just make sure that all downstream code now just has this separation in place.

Reply:
will change